content/review.rst

```diff
@@ -87,12 +90,17 @@ For the person requesting a review
   It is very important to have a good review request message as it will
   help reviewers understand what you have done.
 
-* Check that all changes are covered by automated tests and if they aren't, make sure
-  that the review description contains information regarding why.
+* Check that all changes are covered by automated tests and if they aren't,
+  make sure that the review description contains information regarding why.
 
 * Check the **Reviewer's check list** since those are the things that a
   reviewer will check for sure.
 
+* Before submitting a ticket for review, check that you have documented your
+  work accordingly, for example in the affected repository documentation, or
+  the styleguide.
```

Comment:
Just a suggestion… Some work is documented in Trac's wiki, especially the work done by the system administrators.